samples/import.ts

```diff
@@ -0,0 +1,17 @@
+// Import variants from https://www.typescriptlang.org/docs/handbook/modules.html#import
+
+import { ZipCodeValidator } from "./ZipCodeValidator";
+
+import { ZipCodeValidator as ZCV } from "./ZipCodeValidator";
+
+import * as validator from "./ZipCodeValidator";
+
+import "./my-module.js";
```

Comment:
Is that really valid syntax in TypeScript?

Reply:
Yes this was directly taken from https://www.typescriptlang.org/docs/handbook/modules.html#import

Although they discourage imports only for side effects it is valid.